message MotionWalkMessage{
	required string command 	= 6 	[default = ""];
	repeated float  parameter   	= 7;
}

message MotionHeadMessage{
	required string command 	= 6 	[default = ""];
	repeated float  parameter   	= 7;
}

message MotionActionMessage{
	required string command 	= 6 	[default = ""];
	repeated float  parameter   	= 7;
}
